app/logic/emailHandler.py

```diff
@@ -173,6 +190,7 @@ def send_email(self):
                         # [recipient.email],
                         [self.override_all_mail],
                         email_body,
+                        file_attachment=self.attach_attachments(),

```

Comment:
This is still the old function name

Reply:
resolved

Reply:
it looks like it's still there for some reason